web/src/selectors/Assertion.selectors.ts

```diff
@@ -27,15 +28,20 @@ const selectAffectedSpanList = createSelector(stateSelector, paramsSelector, (st
   return trace?.spans.filter(({id}) => spanIdList.includes(id)) || [];
 });
 
-const AssertionSelectors = () => ({
-  selectAffectedSpanList,
-  selectAttributeList: createSelector(selectAffectedSpanList, spanList => spanList.flatMap(span => span.attributeList)),
-  selectSelectorAttributeList: createSelector(
+const AssertionSelectors = () => {
+  return {
     selectAffectedSpanList,
-    currentSelectorListSelector,
-    (spanList, currentSelectorList) =>
-      SpanAttributeService.getFilteredSelectorAttributeList(spanList.flatMap(span => span.attributeList), currentSelectorList)
-  ),
-});
+    selectAttributeList: createSelector(selectAffectedSpanList, selectAffectedSpans, (spanList, affectedSpans) => spanList.flatMap(span => span.attributeList).concat(SpanAttributeService.additionalAttributes(affectedSpans.length))),
```

Comment:
Let's use the selectAffecedSpans from the wrapping function like SpanSelectors.selectAffectedSpans